draft-ietf-quic-http.md

```diff
@@ -1238,9 +1233,10 @@ MAX_PUSH_ID frame ({{frame-max-push-id}}). A client MUST treat receipt of a
 PUSH_PROMISE frame that contains a larger Push ID than the client has advertised
 as a connection error of H3_ID_ERROR.
 
-A server MUST NOT use the same Push ID in multiple PUSH_PROMISE frames. A client
-MUST treat receipt of a Push ID which has already been promised as a connection
-error of type H3_ID_ERROR.
+A server MAY use the same Push ID in multiple PUSH_PROMISE frames. If so, the
+decompressed request headers MUST be idential.  A client MUST treat receipt of a
```

Comment:
```suggestion
decompressed request headers MUST be identical.  A client MUST treat receipt of a
```

Reply:
maybe a x-ref to the definition of identical would help too

Reply:
Modified to include the same language as on line 605. Hope this helps!